dsl/src/test/kotlin/com/chutneytesting/kotlin/dsl/ChutneyEnvironmentDslTest.kt

```diff
@@ -0,0 +1,19 @@
+package com.chutneytesting.kotlin.dsl
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class ChutneyEnvironmentDslTest {
+
+    @Test
+    fun `Test duplicate taget name in environment`() {
+        val targets: List<ChutneyTarget> = listOf(
+            ChutneyTarget("toto", "url1"),
```

Comment:
Can you add a second non unique target, and at least one unique ?
ChutneyTarget("bou","url3")
ChutneyTarget("bou","url4")
ChutneyTarget("unique","url")

Reply:
Fixed